src/shell/lisa_shell

```diff
@@ -308,19 +319,38 @@ nosetests -v --nocapture --nologcapture \
 }
 
 function lisa-test {
-CMD=${1:-help}
+local iterations=0 # Means use default - see libs/utils/test.py
+local have_more_args=no
+
 echo
-case "x${CMD^^}" in
-"xHELP")
-	_lisa-test-usage
-	;;
-*)
-	_lisa-test $*
+while [[ $# -gt 0 ]]; do
+        case "$1" in
+                help|-h|--help)
+                        _lisa-test-usage
+                        return 0
+                        ;;
+                -n|--wload-iterations)
+                        iterations="$2"
+                        shift
+                        ;;
+                *)
+                        have_more_args=yes
```

Comment:
Although I'd like to keep it as "raw bash" because `getopts` doesn't support longopts like "--iterations". [There's a separate thing called `getopt` but it isn't in OSX etc](https://stackoverflow.com/questions/402377/using-getopts-in-bash-shell-script-to-get-long-and-short-command-line-options) and I guess one day someone might want to get LISA working on Macs. And the raw bash is fine in this case I think.